RealmSwift/Migration.swift

```diff
@@ -0,0 +1,233 @@
+          - A new property without a default was added to an object and not initialized
+            during the migration. You are required to either supply a default value or to
+            manually populate added properties during a migration.
+
+:param: version The current schema version.
+:param: block   The block which migrates the Realm to the current version.
+*/
+public func setDefaultRealmSchemaVersion(schemaVersion: UInt, migrationBlock: MigrationBlock) {
+    RLMRealm.setDefaultRealmSchemaVersion(schemaVersion, withMigrationBlock: accessorMigrationBlock(migrationBlock))
+}
+
+/**
+Specify a schema version and an associated migration block which is applied when
+opening a Realm at the specified path with an old schema version.
+
+Before you can open an existing `Realm` which has a different on-disk schema
+from the schema defined in your object interfaces, you must provide a migration
+block which converts from the disk schema to your current object schema. At the
+minimum your migration block must initialize any properties which were added to
+existing objects without defaults and ensure uniqueness if a primary key
+property is added to an existing object.
+
+You should call this method before accessing any `Realm` instances which
+require migration. After registering your migration block, Realm will call your
+block automatically as needed.
+
+:warning: Unsuccessful migrations will throw exceptions when the migration block is applied.
+          This will happen in the following cases:
+
+          - The migration block was run and returns a schema version which is not higher
+            than the previous schema version.
+          - A new property without a default was added to an object and not initialized
+            during the migration. You are required to either supply a default value or to
+            manually populate added properties during a migration.
+
+:param: version   The current schema version.
+:param: realmPath The path of the Realms to migrate.
+:param: block     The block which migrates the Realm to the current version.
+*/
+public func setSchemaVersion(schemaVersion: UInt, realmPath: String, migrationBlock: MigrationBlock) {
+    RLMRealm.setSchemaVersion(schemaVersion, forRealmAtPath: realmPath, withMigrationBlock: accessorMigrationBlock(migrationBlock))
+}
+
+/**
+Get the schema version for a Realm at a given path.
+:param: realmPath     Path to a Realm file.
+:param: encryptionKey Optional 64-byte encryption key for encrypted Realms.
+:param: error         If an error occurs, upon return contains an `NSError` object
+                      that describes the problem. If you are not interested in
+                      possible errors, omit the argument, or pass in `nil`.
+:returns: The version of the Realm at `realmPath` or `nil` if the version cannot be read.
+*/
+public func schemaVersionAtPath(realmPath: String, encryptionKey: NSData? = nil, error: NSErrorPointer = nil) -> UInt? {
```

Comment:
There is a `RawByte` type, but the Swift stdlib seems to prefer `[UInt8]` in those situations